### ISSUE TITLE
How do we handle the metacrap argument?

### ISSUE THREAD

**Issue by bhaugen** [valueflows/forum.valueflo.ws#88]
I've been meaning to deal with this, but haven't taken the time. Now I am motivated to do so. Anybody else interested? These articles pose a serious challenge to everything we are doing here. Moreover, I have seen them accepted, possibly without reading the original, as a reason to diss the upper case Semantic Web by many programmers, such that in a lot of contexts that I visit regularly, it's impossible to even discuss the concept.

So I think we need to deal with it.

If interested and think you can help, read these articles:
* [The Future of APIs](https://blog.goodapi.co/future-of-apis-c84a76bc9c85#.t6qc39qcx), which I think would accept what we are doing here as useful and necessary.
* [Hacker news discussion of that article](https://news.ycombinator.com/item?id=13211492), which largely, but not universally, disagrees.
* Some articles linked to comments in that discussion attempting to prove the disagreement:
    * [The Semantic Web, Syllogism, and Worldview](http://www.shirky.com/writings/herecomeseverybody/semantic_syllogism.html)
    * [The original statement of the Metacrap argument](http://www.well.com/~doctorow/metacrap.htm)

TL;DR version of the metacrap argument:
> A world of exhaustive, reliable metadata would be a utopia. It's also a pipe-dream, founded on self-delusion, nerd hubris and hysterically inflated market opportunities.

**Comment by almereyda** [comment]
We have moved the ValueFlows organization from GitHub to https://lab.allmende.io/valueflows.

This issue has been closed here, and all further discussion on this issue can be done at

> https://lab.allmende.io/valueflows/forum-valueflo-ws/-/issues/88.

If you have not done so, you are very welcome to register at https://lab.allmende.io and join the ValueFlows organization there.
